src/gsi/calctends_no_ad.f90

```diff
@@ -210,7 +211,7 @@ subroutine calctends_no_ad(st,vp,t,p,mype,u_t,v_t,t_t,p_t,uvflag)
       end do                         
     end if
 
-    call turbl_ad(ges_prsi(1,1,1,it),ges_tv,ges_teta(1,1,1,it),&
+    if(use_pbl)call turbl_ad(ges_prsi(1,1,1,it),ges_tv,ges_teta(1,1,1,it),&
```

Comment:
@jderber-NOAA  a question about why , now use_pbl condition is required to call turbl_ad in calctends_no_ad.f90 and to call turbl_tl in calctends_no.f90  but not used to call turbl in calctends.f90.?  And, seems this change could cause differences in results, right? Will this be able to explain those tiny differences in your regression test?

Reply:
This is one of those things that I started to do, but just didn't have time to finish.  The turbl_ad, turbl_tl and turbl routines all had the first executable line saying if(.not.use_pbl)return.  So the use_pbl in the call for turbl_ad and turbl_tl was redundant and results should not change.

I have completed what I was doing and added a "if (use_pbl) call turbl..." for all the calls to the turbl routines (turbl_tl, turbl_ad, and turbl).  This is added in all of the calctends routines.

Regession tests being rerun.  Results identical to before expected.

Reply:
Thanks for the explanation and clarification!